src/flows/pipes/Schedule/view.tsx

```diff
@@ -228,15 +228,13 @@ const Schedule: FC<PipeProp> = ({Context}) => {
 
     return format_from_js_file(ast)
   }, [queryText, data.interval, data.offset])
-  const hasChanges = useMemo(() => {
-    if (data.task?.id) {
-      return taskText !== data.task.flux
-    }
-    if (data.task.length) {
-      return taskText !== data.task[0].flux
-    }
-    return false
-  }, [taskText, data?.task?.flux])
+  let lastTask
+  if (data.task?.id) {
+    lastTask = data.task
+  } else if (data.task?.length) {
+    lastTask = data.task[0]
```

Comment:
wouldn't this need to be `data.task[data.task.length -1]`? if we're getting the last one?

Reply:
lol i guess that is confusing. i'm referring to the "last created" but it's the first in the list